domain/src/main/kotlin/no/nav/su/se/bakover/domain/brev/BrevInnhold.kt

```diff
@@ -160,6 +160,13 @@ abstract class BrevInnhold {
     ) : BrevInnhold() {
         override val brevTemplate = BrevTemplate.Revurdering.AvsluttRevurdering
     }
+
+    data class InnkallingTilKontrollsamtale(
+        val personalia: Personalia,
+        val saksbehandlerNavn: String,
```

Comment:
Har vi egentlig et behov for navn her? Saksbehandler er vel uansett ikke den bruker har kontakt med når de kommer inn for kontrollsamtale? Ei heller har vi gjort noen form for saksbehandling? Regner med vi ønsker å automatisere dette ganske snart? Kunne vel sannsynligvis bare vært et generelt brev fra NAV?

Reply:
Ja, ikke uenig, men de har bedt om navn i brevet.